centreon/src/Core/Dashboard/Playlist/Application/UseCase/CreatePlaylist/CreatePlaylistValidator.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace Core\Dashboard\Playlist\Application\UseCase\CreatePlaylist;
+
+use Centreon\Domain\Contact\Interfaces\ContactInterface;
+use Core\Dashboard\Application\Repository\ReadDashboardRepositoryInterface;
+use Core\Dashboard\Playlist\Application\Exception\PlaylistException;
+use Core\Dashboard\Playlist\Application\Repository\ReadPlaylistRepositoryInterface;
+
+final class CreatePlaylistValidator
+{
+    /**
+     * @param ReadDashboardRepositoryInterface $dashboardRepository
+     * @param ReadPlaylistRepositoryInterface $playlistRepository
+     */
+    public function __construct(
+        private ReadDashboardRepositoryInterface $dashboardRepository,
+        private ReadPlaylistRepositoryInterface $playlistRepository
+    ) {
+    }
+
+    /**
+     * @param int[] $dashboardIds
+     *
+     * @throws PlaylistException|\Throwable
+     */
+    public function validateDashboardExists(array $dashboardIds): void
+    {
+        $unexistentDashboards = [];
```

Comment:
Typo : the common spelling is "nonexistent", rarely "inexistent", but not "unexistent"